packages/orchestration/src/utils/zoe-tools.js

```diff
@@ -38,25 +38,26 @@ const { assign, keys, values } = Object;
  */
 
 /**
- * @param {Zone} zone
- * @param {{ zcf: ZCF; vowTools: VowTools }} io
+ * @param {ZCF} zcf
+ * @param {VowTools} vowTools
  */
-export const makeZoeTools = (
-  zone,
-  { zcf, vowTools: { retriable, when, allVows, allSettled } },
-) => {
+export const makeZoeTools = (zcf, vowTools) => {
+  const { when, allVows, allSettled, asVow } = vowTools;
```

Comment:
why delay the destructuring? It allows for a mixing of call styles in the same closure.
```js
  return asVow((
```
```js
  return vowTools.aVow(
```